internal/cloudapi/server.go

```diff
@@ -426,27 +426,27 @@ func (h *apiHandlers) Compose(ctx echo.Context) error {
 	var response ComposeResult
 	response.Id = id.String()
 
-	return ctx.JSON(http.StatusOK, response)
+	return ctx.JSON(http.StatusCreated, response)
```

Comment:
also fixed in #1712, but it'll just rebase out

Reply:
Ooops, I should not have commit this.

Reply:
I've cleaned it. I don't want to mix apples and oranges :)